website/pages/[handle]/_/access-tokens.tsx

```diff
@@ -0,0 +1,216 @@
+import Head from 'next/head'
+import { MouseEventHandler, useState } from 'react'
+import toast from 'react-hot-toast'
+import AccessTokenCard from '~/components/access-tokens/AccessTokenCard'
+import Form, { FORM_ERROR } from '~/components/forms/Form'
+import FormButton from '~/components/forms/FormButton'
+import FormInput from '~/components/forms/FormInput'
+import Layout from '~/components/layouts/Layout'
+import Button from '~/components/ui/Button'
+import CopyButton from '~/components/ui/CopyButton'
+import H1 from '~/components/ui/typography/H1'
+import H3 from '~/components/ui/typography/H3'
+import { useAccessTokensQuery } from '~/data/access-tokens/access-tokens-query'
+import { useNewAccessTokenMutation } from '~/data/access-tokens/create-access-token'
+import { useAuth } from '~/lib/auth'
+import { NextPageWithLayout } from '~/lib/types'
+import { NewTokenSchema } from '~/lib/validations'
+
+const ApiTokensPage: NextPageWithLayout = () => {
+  const { refreshSession } = useAuth()
+
+  const [showNewTokenForm, setShowNewTokenForm] = useState(false)
+  const [newToken, setNewToken] = useState('')
+
+  const {
+    mutateAsync: createNewAccessToken,
+    isLoading: creatingNewAccessToken,
+  } = useNewAccessTokenMutation({
+    onSuccess() {
+      toast.success('Successfully created token!')
+    },
+  })
+
+  const createNewToken = async ({ tokenName }: { tokenName: string }) => {
+    try {
+      await refreshSession()
```

Comment:
What's the reason we need to refresh the session before creating an access token?

Reply:
Good catch, not needed, removed.